tests/test_scalar_comprehensive.py

```
@@ -0,0 +1,495 @@
+        self.assertTrue(np.allclose(s137.vals, [3.5, 2.5]))
+
+        # Test sort with axis
+        s138 = Scalar([[3., 1., 4.], [2., 5., 1.]])
+        s139 = s138.sort(axis=1)
+        self.assertTrue(np.allclose(s139[0].vals, [1., 3., 4.]))
+
+        # Test solve_quadratic with complex roots (should mask)
+        a3 = Scalar(1.)
+        b3 = Scalar(1.)
+        c3 = Scalar(1.)
+        x0_3, x1_3 = Scalar.solve_quadratic(a3, b3, c3)
```

Ruff (0.14.7)

Unpacked variable `x0_3` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)